scarb/src/ops/resolve.rs

```diff
@@ -78,10 +89,28 @@ pub fn generate_compilation_units(
                         member.id, pkg.id.name
                     ));
                 }
+
+                let package_version = pkg.manifest.metadata.cairo_version.clone();
+                if package_version.is_some() {
+                    let package_version = package_version.unwrap();
```

Comment:
```suggestion
                if let Some(package_version) = package_version  {
```
FYI https://doc.rust-lang.org/rust-by-example/flow_control/if_let.html